apps/website/src/pages/docs/button.mdx

```diff
@@ -18,51 +18,92 @@ import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='buttons-button' />
-
-<LiveExample src='Button.main.tsx' truncate={false}>
+<LiveExample src='Button.main.tsx'>
   <AllExamples.ButtonMainExample client:load />
 </LiveExample>
 
-Buttons are the recipients of a user onClick action (a mouse click for browsers and a finger press on touch-sensitive devices). Buttons have states and events and should be used when a form is submitted (post) or where the button represents a clear call to action on a page. In some cases, buttons can also be used in place of href links to draw special attention to that action.
+## Appearance
+
+### Default
 
-## Button
+The default button should be used in most circumstances. If you are unsure as to what button to use, use this button!
 
 <LiveExample src='Button.default.tsx'>
   <AllExamples.ButtonDefaultExample client:load />
 </LiveExample>
 
-### Props
+### High visibility
+
+This is the button to emphasize an action applicable to an area of the page or a [dialog](dialog). High visibility buttons are used to draw the user’s attention to the main action of the page.
+There should only be one high visibility button per area of the user interface.
+
+<LiveExample src='Button.highvisibility.tsx'>
+  <AllExamples.ButtonHighVisibilityExample client:load />
+</LiveExample>
+
+### Call to action
+
+The call-to-action button, also referred to as the CTA, is the button that is used for most full page layouts and whole-page overlays (think Next in stepper-like interactions). This is the biggest and most noticeable button.
+One call-to-action button per visible page is the rule. Treat it as a large, “CLICK HERE” guide for the user.
+
+<LiveExample src='Button.cta.tsx'>
+  <AllExamples.ButtonCTAExample client:load />
+</LiveExample>
+
+### Borderless
+
+The borderless button is useful in tighter areas where using a default button would visibly cause too many boxes.
+
+The borderless button is often used by actionable icons in relation to another component.
+They are commonly used within [table](table) rows and widgets. A good example would be the close icon within a modal, toast notification, or alert.
```

Comment:
Should there be links to Modal, Toast, and Alert pages?

Reply:
Removed this part as we are handling close buttons for the user.